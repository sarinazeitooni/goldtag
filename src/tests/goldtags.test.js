import React from 'react';
import {shallow} from 'enzyme';
import Goldtags from "../components/goldtags/goldtags";
describe("goldtags component" , ()=>{
    it("should match snap shot" , ()=>{
        const wrapper = shallow(<Goldtags/>);
        expect(wrapper).toMatchSnapshot();
    });
    it("should have tags container" , ()=>{
        const wrapper = shallow(<Goldtags/>);
        const container = wrapper.find('div.cards-container');
        expect(container.exists()).toBe(true);
    })
})