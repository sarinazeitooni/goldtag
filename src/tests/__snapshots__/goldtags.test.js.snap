// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`goldtags component should match snap shot 1`] = `ShallowWrapper {}`;
